### DISCUSSION TITLE
Symfony Cloud: delete cloud data, but don't want to delete the project ..

### DISCUSSION THREAD

**Discussion by htmlshaman1** [symfony/symfony#44288] [Q&A] [answered]
Hi Everyone!

I'm re-using some code, and I have some Symfony cloud data somewhere in my project. I don't want to just run delete:project because I'll take down a live site.  

Thanks,

**Comment by stof** [comment, marked as answer]
for support for commercial services, please contact the dedicated support: https://symfony.com/support

**Comment by htmlshaman1** [comment]
Thanks stof, will do!
